goal_src/jak1/pc/progress-pc.gc

```diff
@@ -1182,6 +1208,209 @@
                   (reset-original-camera *pc-settings*)
                   (sound-play "cursor-options")
                   )
+                 ;; LEVEL SELECT
+                 (((game-text-id training-level-name))
+                  (when (-> *pc-settings* speedrunner-mode?)
+                    (set! *speedrun-setup-function* (lambda ()
+                      ;; extra setup for IL speedruns
+                      (speedrun-set-common-settings)
+                      ))
+                    )
+                  (initialize! *game-info* 'game (the-as game-save #f) "game-start")
+                  )
+                 (((game-text-id village1-level-name))
+                  (when (-> *pc-settings* speedrunner-mode?)
+                    (set! *speedrun-setup-function* (lambda ()
+                      ;; give enough orbs to buy all cells 120+120+90+90=420
+                      (set! (-> *game-info* money-total) 420.0)
+                      (set! (-> *game-info* money) 420.0)
+                      ;; extra setup for IL speedruns
+                      (speedrun-set-common-settings)
+                      ))
+                    )
+                  (initialize! *game-info* 'game (the-as game-save #f) "village1-hut")
+                  )
+                 (((game-text-id beach-level-name))
+                  (when (-> *pc-settings* speedrunner-mode?)
+                    (set! *speedrun-setup-function* (lambda ()
+                      ;; unlock blue eco vent
+                      (close-specific-task! (game-task jungle-eggtop) (task-status need-resolution))
+                      (send-event *target* 'get-pickup (pickup-type fuel-cell) (the float (game-task jungle-eggtop)))
```

Comment:
Just closing the task itself should suffice, but its probably "clearer" this way - nice call

Reply:
Picking up the collectable (not just closing the task) seems to be necessary from my testing

Reply:
I can confirm from my testing but it does look like it might be possible to just to `  (send-event *target* 'get-pickup (pickup-type fuel-cell) (the float (game-task jungle-eggtop)))` without closing the task?

The eco vent turned on for me, but I also crashed on a save/load but I was on an unstable modified build